client/src/components/Header/Header.scss

```diff
@@ -4,15 +4,153 @@
     url('../../assets/fonts/WTT-THIN.ttf') format('truetype');
 }
 
+// .header {
```

Comment:
I think SASS works for this. But IIRC, you tried styled components first. IF you do want to implement this with styled components, I think this reference would help write styled components in a more concise and reusable way: https://css-tricks.com/dry-ing-up-styled-components/

Basically, styled components are designed to be reusable and modular by doing some good old-fashioned inheritance

Reply:
I'm open to having this discussion. Its just hard for me as I don't really know what conventions there are for styled components especially when it comes to using it for page structuring and being imported and being reused constantly

Reply:
Okay, we can have that discussion on styled components vs SCSS on Wednesday. If we do go with styled components, we can go through some examples and establish a convention for the project